ynput/ayon-core
# Maya: Prompt reset scene context on saving to another task

## Changelog Description

Add a prompt to reset workfile data (frame_range, fps, resolution, colorspace) or publish instances (target folder+task) to the folder or task being saved to.

## Additional info

![image](https://github.com/ynput/ayon-core/assets/2439881/39c5d40e-f3a5-4fe1-950b-70645ea0f9c9)

I'll be preparing a similar matching feature for Houdini

## Additional info

This is the Maya equivalent of #259 

## Testing notes:

1. Open a workfile
2. Save it to another folder or task via workfile tool
3. The prompt should appear and be clear what it does.
